docs-importer/main.js

```diff
@@ -142,13 +143,32 @@ const askPath = () => {
 
         recursive(answers.path, function (error, files) {
           spinner.succeed(files.length + ' files in this directory');
-          askTag();
+          askFileFilter();
         });
       });
     });
   });
 };
 
+// Ask for the file filter
+const askFileFilter = () => {
+  console.log('');
+
+  inquirer.prompt([
+    {
+      type: 'input',
+      name: 'fileFilter',
+      message: 'What pattern do you want to use to match files? (eg. *.+(pdf|txt|jpg) - defaults to *)',
+      default: prefs.importer.fileFilter || "*"
+    }
+  ]).then(answers => {
+    // Save fileFilter
+    prefs.importer.fileFilter = answers.fileFilter  ? answers.fileFilter : '*';
```

Comment:
```suggestion
    prefs.importer.fileFilter = answers.fileFilter;
```
value is set in L162

Reply:
Makes complete sense, amended the previous commit.